scripts/createYOEOLabelsFromTORSO-21.py

```diff
@@ -239,9 +213,14 @@ def range_limited_float_type_0_to_1(arg):
 with open(test_path, "w") as test_file:
     test_file.writelines([str(os.path.join(destination_dir, image_name)) + "\n" for image_name in test_images])
 
-names_path = os.path.join(destination_dir, "yoeo.names")
+# The names file contains the class names of bb detections and segmentations
+names_path = os.path.join(destination_dir, "yoeo.names.yaml")
+names = {
+    'detection': CLASSES['bb_classes'],
+    'segmentation': ['background', 'lines', 'field']
```

Comment:
Maybe add the original names here.

Reply:
Afaik segmentations in torso are not named either way, but they are in yoeo. So we can adjust the names on the top and add them here.

Reply:
okay, I was not quite sure on this